src/extension.ts

```diff
@@ -69,7 +69,6 @@ export async function activate(context: vscode.ExtensionContext) {
 	);
```

Comment:
You might want to add something like 

```
  const databricksExtension: vscode.Extension<any> = vscode.extensions.getExtension("databricks.databricks");
  if (databricksExtension) {
    await databricksExtension.activate();
  }
 ```
 
to the top of the `activate` function to guarantee the ordering of extensions. Otherwise panels might shuffle around depending on which extension gets activated first.

Reply:
that piece of code is in the ConnectionManager and is only executed if we are actually using the new DatabricksConnectionManagerDatabricks class